packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -486,6 +489,113 @@ $Calendar-dayPicker-margin: 10px;
     }
   }
 
+  // Show the month picker over date picker (1 column layout), if enabled
```

Comment:
It looks like updates from another pull request are also here, I'm going to sort it out, sorry!